packages/neuron-wallet/src/controllers/multisig.ts

```diff
@@ -12,6 +11,7 @@ import OfflineSignService from '../services/offline-sign'
 import Multisig from '../models/multisig'
 import SystemScriptInfo from '../models/system-script-info'
 import NetworksService from '../services/networks'
+import { config as lumosConig, helpers, utils, config } from '@ckb-lumos/lumos'
```

Comment:
What's the difference between `config as lumosConfig` and `config`

Reply:
setting `lumosConfig` as alias for `config`, because there is another viarable name `config` in the scope

Reply:
I didn't get it, `lumosConfig` and `config` are both from `@ckb-lumos/lumos.config`, pointing to the same one.

BTW, `lumosConig` is a typo

Reply:
https://github.com/nervosnetwork/neuron/blob/b4f7de26155e09a4ce7cab0ad5364664c765ac1f/packages/neuron-wallet/src/controllers/multisig.ts#L110-L117

`config` is delared here in line 110, if `import { config } from '@ckb-lumos/lumos'`, then there are two `config`s in the mapFunction, I'll fix the typo

Reply:
```suggestion
import { config as lumosConig, helpers, utils } from '@ckb-lumos/lumos'
```